website/views.py

```diff
@@ -241,6 +242,10 @@ def resolve_guid(guid, suffix=None):
         if not referent.deep_url:
             raise HTTPError(http.NOT_FOUND)
         if isinstance(referent, PreprintService):
+            if re.match('download/?', suffix) and referent.primary_file:
+                if request.args.get('format'):
+                    return redirect('{}/export?format={}&url={}'.format(MFR_SERVER_URL, request.args['format'], referent.primary_file.generate_waterbutler_url()))
```

Comment:
The URL needs to use the `urllib.url_encode`? equivalent of JS' `encodeURIComponent`